doc/api/assert.md

```diff
@@ -256,19 +256,20 @@ If the values are not equal, an `AssertionError` is thrown with a `message`
 property set equal to the value of the `message` parameter. If the `message`
 parameter is undefined, a default error message is assigned.
 
-## assert.fail(message)
+## assert.fail([message])
 ## assert.fail(actual, expected, message, operator)
 <!-- YAML
 added: v0.1.21
 -->
 * `actual` {any}
 * `expected` {any}
-* `message` {any}
+* `message` {any} (default: 'Failed')
 * `operator` {string} (default: '!=')
 
 Throws an `AssertionError`. If `message` is falsy, the error message is set as
 the values of `actual` and `expected` separated by the provided `operator`.
 Otherwise, the error message is the value of `message`.
+If you provide no arguments at all, a default message will be used instead.
```

Comment:
If it helps at all, according to [our style guide](https://github.com/nodejs/node/blob/f803e77b1e2347dbce9d8098f3b6e231f4ce5fcd/doc/STYLE_GUIDE.md):

> * Generally avoid personal pronouns in reference documentation ("I", "you", "we").
>   * Pronouns are acceptable in more colloquial documentation, like guides.
>   * Use gender-neutral pronouns and mass nouns. Non-comprehensive examples:
>     * OK: "they", "their", "them", "folks", "people", "developers", "cats"
>     * NOT OK: "his", "hers", "him", "her", "guys", "dudes"

The style guide has had the "generally avoid" part since June 2015 (when it lived in the docs repo where its visibility was unfortunately limited). https://github.com/nodejs/docs/blob/753d262f1530b1aff3c7e46d3c116d810d1698d3/GUIDE.md

It does seem like a "why" for it might be beneficial in the style guide. Maybe @nodejs/documentation can sort out the best reasons. For me, seeing "you" in technical documentation makes me suspicious that it was written carelessly. I immediately wonder if its out of date, inaccurate, contains omissions, etc. That may or may not be a sensible reaction on my part, but there it is.

The other option, of course, would be to change the style guide to say we're perfectly fine with pronouns like "you" etc. in our technical docs.